### DISCUSSION TITLE
Batteries discharging fast

### DISCUSSION THREAD

**Discussion by Patrick257** [PiSupply/PiJuice#925] [Q&A]
Hello,

After first having been delivered a faulty PiJuice, I'm now quite happy with the second shipment. Based on the PiJuice, a Raspberry PI 4B and several further connected devices (USB spealer, several HATS like a motor controller, a microphone array, and multiple sensors) I'm building a home robot project.

The larger batteries that can be purchased with the PiJuice (ie. 5000 and 12000 mAh) come with rather short wires. For my installation, I had to extend the length of the wires to be able to safely attach the battery and connect it with the screw terminal.

After a first experiment with the 5000 mAh battery and a self-soldered wire extension, I wasn't happy at all because I was getting random readings in the PiJuice GUI and very short battery runtimes. I decided to redo the connection with possibly better soldering and now I use the 12000 mAh battery. This time, I have tested with a multimeter that the full voltage is available at the screw terminal. I.e. when the battery was just unboxed, I measurd 3.7ish volts consistently.

However, I still have the problem that the battery discharges amazingly fast. Specifically, I get a runtime of about 1-2 hours from 100% to about 20% (I have configured the shutdown at 20% low battery). From my point of view, even with the power hungry 4B and even with some peripherals like the ones mentioned above, the runtime should be much longer?

I have then connected a USB multimeter between the charger and the PiJuice micro USB input. It measures a draw of around 1 A when idling and around 1.6-1.8 with some activity.

I would have expected the 12000 mAh battery to last for 6-12 hours based on the current draws measured above.

Anyone has any hint to point me in the right direction what I should look into? I'm not an electronics professional - I've just been lucky so far that everything else in building the robot has worked out amazingly well. ;-)

Thanks for considering and best regards

Patrick

**Comment by tvoverbeek** [comment]
From the main PiJuice README:
Output amperage - maximum current at 5V GPIO is 2.5A and at VSYS output 2.1A, but also this depends heavily on battery capacity. For BP7X have measured around 1.1A at 5V GPIO and around 1.6A at VSYS output. Obviously this also depends heavily on the current draw demanded by the Raspberry Pi / device itself. To achieve maximum of 2.5A it will need battery over 3500mAh.

The limit is due to the voltage regulator which produces the 5V.
A power hungry 4B = peripherals like motor controllers (if they are onlu powered by the RPi) will be certainly too much.
In such power hungry applications the PiJuice should only be used for a safe shutdown when external power disappears.
If you need to power a complete robot you need a different power solution.

**Reply by Patrick257** [reply]
There might be one misunderstanding: The motor controller has a separate power source. It sits on top of the RPi as a hat, so I assume it will consume SOME current from the Pi, but not that much. As mentioned, the power draw via the USB multimeter does not seem that much - at 1.0-1.8 A, it is still quite far away from the max 2.5. I specifically bought the larger batteries based on the Readme you posted above.

So I'm irritated due to the measured power draw and the "perceived" draw based on the time it takes for the large battery to discharge. The time of 1-2 hours would indicate a power draw of 6-12 A, right? There are certainly no peripherals attached that draw that much current.

**Comment by tvoverbeek** [comment]
You cannot expect to get 6A at 5V from the 12000 mAh battery. Battery capacity usually assumes the nominal battery voltage of 3.7V.
The efficiency of the power converters between the battery and the output is ca 75%.
Due to internal resistance and battery condition the delivered power will be smaller than the theoretical value.
That is all I can say. I am not employed by Pi-Supply, just a volunteer.
Contact Pi-Supply directly if you bought it there.

**Reply by Patrick257** [reply]
Thanks, Ton, for following up (no expectations here, I understand this is not a PiSupply support, just looking if maybe someone experienced similar issues or has a broader range of experience). I do not expect 6A from the battery though, my point was only that if the 12000 mAh battery is empty after 1-2 hours, that's what it "looks like" mathematically. And then I just don't know where those 6 A would go (certainly not into the RPi). Even considering the fact that the delivered power will be lower than the theoretical value as you point out, 1-2 hours of runtime on a 12000 mAh battery seem very, very low for the use case.
